tidb-retool-integration.md

```diff
@@ -0,0 +1,22 @@
+# TiDB and Retool Integration
+1. Get TiDB Details & Whitelist Retool IP Address
```

Comment:
Could you first explain what Retool is? What users can do with it? The scenario?